### PR TITLE
fix: reverse -> descending

### PR DESCRIPTION
**Pull request by eitsupi**
A follow up for #291

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ for demonstration purposes (#240).
 - `<DataFrame>$glimpse()` is a fast `str()`-like view of a `DataFrame` (#277).
 - `$over()` now accepts a vector of column names (#287).
 - Replace the argument `reverse` by `descending` in all sorting functions. This
-is for consistency with the upstream Polars (#290).
+  is for consistency with the upstream Polars (#290, #293).
```
Comment (etiennebacher) [inline NEWS.md:20]:
```suggestion
  is for consistency with the upstream Polars (#291, #293).
```

**Comment by etiennebacher** [comment]
Thanks, I don't understand how #291 passed the tests with these changes missing...

**Comment by eitsupi** [comment]
> Thanks, I don't understand how #291 passed the tests with these changes missing...

I think this is because it does not actually use the names of the arguments on the Rust side, but only the order of the arguments.

**Review by etiennebacher — APPROVED** [review]
thanks!

**Pull request merged**